ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -224,6 +229,12 @@ const BoxHeader = ({text= '', header =  '', subHeader = '', style= {}, ...props}
   </div>;
 };
 
+interface DataSetPreviewList {
+  domain: Domain;
+  isLoading: boolean;
+  values?: Array<DataSetPreviewValueList>;
+}
+
 interface Props {
```

Comment:
nit: can we change the name of Props and state as per the naming conventions DataSetProps, data setstate

Reply:
I thought the convention was to name it Props, and then export as `DataSetPageProps`? Is that not the case?

Reply:
Had a chat about this offline, we are going with the way this is.